Conductor/Api/IHumanTaskResourceApi.cs

```diff
@@ -0,0 +1,468 @@
+﻿using Conductor.Client;
+using Conductor.Client.Models;
+using System;
+using System.Collections.Generic;
+
+namespace conductor_csharp.Api
+{
+    /// <summary>
+    /// Represents a collection of functions to interact with the API endpoints
+    /// </summary>
+    public interface IHumanTaskResourceApi : IApiAccessor
+    {
+        #region Synchronous Operations
+        /// <summary>
+        /// Claim a task to an external user
+        /// </summary>
+        /// <remarks>
+        /// 
+        /// </remarks>
+        /// <exception cref="ApiException">Thrown when fails to make API call</exception>
+        /// <param name="taskId"></param>
+        /// <param name="userId"></param>
+        /// <param name="overrideAssignment"> (optional, default to false)</param>
+        /// <returns>HumanTaskEntry</returns>
+        HumanTaskEntry AssignAndClaim(string taskId, string userId, bool? overrideAssignment = null);
+
+        /// <summary>
+        /// API for backpopulating index data
+        /// </summary>
+        /// <remarks>
+        /// 
+        /// </remarks>
+        /// <exception cref="ApiException">Thrown when fails to make API call</exception>
+        /// <param name="_100"></param>
+        /// <returns>Dictionary&lt;string, Object&gt;</returns>
+        Dictionary<string, Object> BackPopulateFullTextIndex(int? _100);
```

Comment:
we can remove this API -- this is only for some maintenance

Reply:
removed